crates/rolldown/tests/rolldown/errors/unresolved_import/unknown_module_type_entry/artifacts.snap

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
The test could not passed in CI machine, can you use a universal path, or just revert it.

Reply:
It doesn't seem like this problem has occurred?

Reply:
https://github.com/rolldown/rolldown/actions/runs/10265126020/job/28400667544?pr=1866#:~:text=created%20previously%20unseen%20snapshot%20/home/runner/work/rolldown/rolldown/crates/rolldown/tests/rolldown/errors/unresolved_import/unknown_module_type_entry/artifacts.snap, this should be a bug related to our test infra, cc @hyf0

Reply:
![image](https://github.com/user-attachments/assets/0b25daa1-3041-48b5-a566-a8071f13567c), snapshot got updated without confirm

Reply:
Got it. Let me fix it.

Reply:
I understand, is it due to the issue of absolute path?
<img width="665" alt="image" src="https://github.com/user-attachments/assets/822ae6f4-78de-485f-bab8-76ead5cd4259">

Reply:
> I understand, is it due to the problem of absolute path? <img alt="image" width="665" src="https://private-user-images.githubusercontent.com/165626830/355424997-822ae6f4-78de-485f-bab8-76ead5cd4259.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI5NDI5MDEsIm5iZiI6MTcyMjk0MjYwMSwicGF0aCI6Ii8xNjU2MjY4MzAvMzU1NDI0OTk3LTgyMmFlNmY0LTc4ZGUtNDg1Zi1iYWI4LTc2ZWFkNWNkNDI1OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgwNlQxMTEwMDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03YjFjMDdjNDYyM2E4YzJmYTA5M2M1ZjI0MWQ3ZWRhNGI2MDZiZGM5ZjhmYjQ3ZWQ4YWZmN2Q4NjdjODQ3OWM3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.taSc_cRXqkEFkhAkUATYktBvuyWiLSb9MOpH5VZsMBk">

Yeah, should use relative path instead.

Reply:
Done at https://github.com/rolldown/rolldown/pull/1878.